src/components/TextField/TextField.tsx

```diff
@@ -1,31 +1,21 @@
 import classNames from 'classnames';
 import React, { useState } from 'react';
-
-type Props = {
-  name: string,
-  value: string,
-  label?: string,
-  required?: boolean,
-  onChange?: (newValue: string) => void,
-};
-
-function getRandomDigits() {
-  return Math.random().toString().slice(2);
-}
+import { getRandomDigits } from '../../helpers/getRandomDigits';
+import { Props } from './TextField.types';
 
 export const TextField: React.FC<Props> = ({
   name,
   value,
   label = name,
   required = false,
   onChange = () => {},
+  isValidUrl = true,
```

Comment:
```suggestion
  isValidValue = true,
```

The field is a universal component, so validation must be universal too